tests/devices/test_default_qutrit.py

```diff
@@ -260,6 +260,292 @@ def test_apply_rotations_one_wire(self, qutrit_device_1_wire, tol):
+
+    def test_expval_estimate(self):
+        """Test that the expectation value is not analytically calculated"""
+
+        dev = qml.device("default.qutrit", wires=1, shots=3)
+
+        @qml.qnode(dev)
+        def circuit():
+            return qml.expval(qml.THermitian(np.array([[0, 1, 0], [1, 0, 0], [0, 0, 0]]), wires=0))
+
+        expval = circuit()
+
+        # With 3 samples we are guaranteed to see a difference between
+        # an estimated expectation value and an analytically calculated one
+        assert not np.isclose(expval, 0.0)
+
+
+class TestVar:
+    """Tests that variances are properly calculated."""
+
+    # TODO: Add test for var of non-parametrized observables
+
+    @pytest.mark.parametrize(
+        "observable,state,expected_output,mat",
+        [
+            (qml.THermitian, [1, 0, 0], 1, [[1, 1j, 0], [-1j, 1, 0], [0, 0, 1]]),
+            (qml.THermitian, [0, 1, 0], 1, [[1, 1j, 0], [-1j, 1, 0], [0, 0, 1]]),
+            (
+                qml.THermitian,
+                [1 / math.sqrt(3), -1 / math.sqrt(3), 1j / math.sqrt(3)],
+                2 / 3,
+                [[1, 1j, 0], [-1j, 1, 0], [0, 0, 1]],
+            ),
+        ],
+    )
+    def test_var_single_wire_with_parameters(
+        self, qutrit_device_1_wire, tol, observable, state, expected_output, mat
+    ):
+        """Tests that variances are properly calculated for single-wire observables with parameters."""
+
+        obs = observable(np.array(mat), wires=[0])
+
+        qutrit_device_1_wire.reset()
+        qutrit_device_1_wire._state = np.array(state).reshape([3])
+        qutrit_device_1_wire.apply([], obs.diagonalizing_gates())
+        res = qutrit_device_1_wire.var(obs)
+
+        assert np.isclose(res, expected_output, atol=tol, rtol=0)
+
+    A = np.array([[1, 0, 0], [0, -1, 0], [0, 0, 2]])
+    B = np.array([[4, 0, 0], [0, -2, 0], [0, 0, 1]])
+    obs_1 = np.kron(A, B)
+
+    C = np.array([[0, -1j, 0], [1j, 0, 0], [0, 0, 0]])
+    D = np.array([[1, 2, 3], [2, 1, 3], [3, 3, 2]])
+    obs_2 = np.kron(C, D)
+
+    @pytest.mark.parametrize(
+        "observable,state,expected_output,mat",
+        [
+            (
+                qml.THermitian,
+                [1 / math.sqrt(3), 0, 1 / math.sqrt(3), 1 / math.sqrt(3), 0, 0, 0, 0, 0],
+                10.88888889,
+                obs_1,
+            ),
+            (
+                qml.THermitian,
+                [0, 0, 0, 0, 0, 0, 0, 0, 1],
+                0,
+                obs_1,
+            ),
+            (
+                qml.THermitian,
+                [0.5, 0, 0, 0.5, 0, 0, 0, 0, 1 / math.sqrt(2)],
+                9,
+                obs_1,
+            ),
+            (
+                qml.THermitian,
+                [0, 0, 1 / math.sqrt(2), 1 / math.sqrt(2), 0, 0, 0, 0, 0],
+                18,
+                obs_2,
+            ),
+            (
+                qml.THermitian,
+                np.array([1 / 3] * 9),
```

Comment:
Any particular reason this one is `np.array`? (Occurs in the previous set of tests as well)

Reply:
I don't see why it needs to be a numpy array, so I'll replace it with just the list.